raiden/network/transport/matrix/client.py

```diff
@@ -833,15 +825,10 @@ def _handle_responses(self, currently_queued_responses: List[JSONResponse]) -> N
                     event["room_id"] = room_id
                     room._put_event(event)
 
-                all_messages.append(
-                    (
-                        room,
-                        [
-                            message
-                            for message in sync_room["timeline"]["events"]
-                            if message["type"] == "m.room.message"
-                        ],
-                    )
+                all_messages.extend(
+                    message
+                    for message in sync_room["timeline"]["events"]
+                    if message["type"] == "m.room.message"
```

Comment:
actually there should not be any messages arriving through rooms anymore. I have no strong feelings if we should leave adding the messages to `all_messages`

Reply:
Hm, good point, I'll remove it.